account_reconcile_oca/tests/common.py

```diff
@@ -0,0 +1,85 @@
+from odoo import Command
+
+from odoo.addons.account_reconcile_model_oca.tests.common import (
+    TestAccountReconciliationCommon as TestAccountReconciliationModelCommon,
+)
+
+
+class TestAccountReconciliationCommon(TestAccountReconciliationModelCommon):
```

Comment:
Don't move this to a new file, but keep it in the same one with 2 classes for reducing the diff.

Reply:
ok, apparently, it's fine. I merged it back in same file :)